### PR TITLE
VACMS-7662: Add field_body text format batching script.

### LINKED ISSUE #7662: Create script to force save 'limited html' on all system service descriptions.
As a developer I need a script to force save the value of regional_health_care_service_des nodes  field_body.allowed_formats to  "rich_text_limited"

## AC
- [ ] script should reside in /scripts/content/VACMS-76626-node-regional_health_care_service_des-force-limited-rich-text.php
- [ ] Should be self-batching similar in structure to /scripts/content/VACMS-6456-remove-leading-care-we-provide-h3-tags.php
- [ ] Set a revision log messsage of "Saved to set body description to 'limited rich text."
- [ ] Should run as CMS Migrator.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
closes #7662

## Testing done
Visual after script run

## Screenshots
A one off that I tested by limiting the query to node 39768:
image

Here's how I limited:
image

## QA steps
 - [x] run `lando drush scr scripts/content/VACMS-7662-node-regional_health_care_service_des-force-limited-rich-text.php`
 - [x] visit `admin/content` and altered nodes will appear first
 - [x] visit a handful, visually verifying correct revision create (cms migrator) and messages, as well as field_body being set to Rich text limited

### CODE REVIEW

File: scripts/content/VACMS-7662-node-regional_health_care_service_des-force-limited-rich-text.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+function va_gov_force_save_limited_html_on_service_descriptions(array &$sandbox) {
+  $node_storage = \Drupal::entityTypeManager()->getStorage('node');
+
+  // Get the node count for VAMC System Health Service nodes.
+  // This runs only once.
+  if (!isset($sandbox['total'])) {
+    $query = Drupal::database()->select('node_field_data', 'nfd');
+    $query->join('node__field_body', 'nfb', 'nfd.nid = nfb.entity_id');
+    $query->fields('nfd', ['nid']);
+    $query->condition('nfd.nid', 39768);
```
Comment (swirtSJW) [inline scripts/content/VACMS-7662-node-regional_health_care_service_des-force-limited-rich-text.php:37]: Why are we using this node id?  It equates to https://prod.cms.va.gov/boston-health-care/health-services/adaptive-sports-at-va-boston-health-care

**Review by swirtSJW — COMMENTED** [review]
Looks good.  Just the one question.

File: scripts/content/VACMS-7662-node-regional_health_care_service_des-force-limited-rich-text.php (hunk 1 above)
Comment (swirtSJW) [reply scripts/content/VACMS-7662-node-regional_health_care_service_des-force-limited-rich-text.php:37]: Nevermind. I see, that was how you limited it to one.    You can remove this so it runs them all.

Comment (ethanteague) [reply scripts/content/VACMS-7662-node-regional_health_care_service_des-force-limited-rich-text.php]: Ugh. Thought I force pushed it last night, but here my file sits with the unsaved dot atop. Sorry about that, @swirtSJW 
image

**Review by swirtSJW — APPROVED** [review]
Looks great.
I ran this on tugboat
image

It completed 
image

Logging all looks good with no funky errors or notices 
image

Revision logs messages look good
image

**Pull request merged**